ts/features/bonus/cgn/utils/filters.ts

```diff
@@ -27,76 +32,73 @@ export type Category = {
   colors: ComponentProps<typeof LinearGradient>["colors"];
 };
 
+export const getGradientColorValues = (
```

Comment:
Maybe this function could become a global utility for the `IOColorGradients`, what do you think?